packages/engine/Source/Core/ITwinPlatform.js

```diff
@@ -38,54 +38,19 @@ ITwinPlatform.ExportType = Object.freeze({
 });
 
 /**
- * Types of Reality data
+ * Types of Reality data. This is a partial list of types we know we can support
```

Comment:
Why are we dropping the unsupported types here?
In the loading functions, we already create a local `supportedRealityDataTypes` array, so it seems like this enum could still retain all valid types, including unsupported ones.

Reply:
This was requested by the iTwin team. The types are potentially changing and/or not a complete list so I trimmed this down to only those types we really care about. I still wanted to keep it in the enum instead of only in the array of `supportedRealityTypes` to make it centrally located and easy to change later.